src/EnergyPlus/DaylightingManager.hh

```diff
@@ -82,6 +82,8 @@ namespace Dayltg {
     constexpr int octreeCrossover(100); // Octree surface count crossover
     constexpr int NTH(18);              // Number of azimuth steps for sky integration
     constexpr int NPH(8);               // Number of altitude steps for sky integration
+
+    // It's crazy having both NPH and NPHMAX
```

Comment:
I had a diff here which took a little while to find. Why do we have both of these?